community-operators/synopsys/synopsys-operator.2019.1.0.yaml

```diff
@@ -0,0 +1,288 @@
+    mediatype: image/png
+  version: 2019.1.0
+  maturity: stable
+  maintainers:
+  - name: Synopsys, Inc
+    email: software-integrity-support@synopsys.com
+  provider:
+     name: Synopsys, Inc
+  labels:
+     alm-owner-etcd: synopsys-operator
+     operated-by: synopsys-operator
+  selector:
+     matchLabels:
+       alm-owner-etcd: synopsys-operator
+       operated-by: synopsys-operator
+  links:
+   - name: Synopsys Integrity
+     url: https://www.synopsys.com/software-integrity.html
+   - name: OpsSight Data Sheet
+     url: https://www.synopsys.com/content/dam/synopsys/sig-assets/datasheets/openshift-ds-ul.pdf
+   - name: Synopsys Operator Documentation
+     url: https://synopsys.atlassian.net/wiki/spaces/BDLM/pages/34373652/Synopsys+Operator
+   - name: Synopsys Operator Source Code
+     url: https://github.com/blackducksoftware/synopsys-operator
+  installModes:
+   - type: OwnNamespace
+     supported: false
```

Comment:
OwnNamespace should be set to true if your operator watches all namespaces.
@alecmerdler

Reply:
fixed in latest push